.github/workflows/vercel-deploy.yml

```
@@ -0,0 +1,81 @@
+          COUNT=0
+          until curl -s -o /dev/null -w "%{http_code}" "$DEPLOY_URL" | grep -q "200\|404"; do
+            if [ $COUNT -eq $MAX_RETRIES ]; then
+              echo "Deployment verification failed after $MAX_RETRIES attempts"
+              exit 1
+            fi
+            COUNT=$((COUNT+1))
+            sleep 10
+          done
+
+      - name: Comment PR
+        uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)